src/js/themes/darkTheme/colors.ts

```diff
@@ -39,9 +39,11 @@ export const colors: ThemeColors = {
   },
   primary: {
     default: '#1098FC',
+    defaultHover: '#0092FA',
```

Comment:
Matches figma and has css in js and css variable token
![Screenshot 2024-01-18 at 5 56 07 PM](https://github.com/MetaMask/design-tokens/assets/8112138/4a0fa6d6-e6e1-49dc-a232-f0b97386c146)